package.json

```diff
@@ -71,9 +69,13 @@
     "time-grunt": "1.2.1"
   },
   "scripts": {
-    "test": "grunt test; grunt lint",
-    "start": "node --harmony-arrow-functions index.js",
-    "startdev": "iojs --harmony-arrow-functions index.js --development -c"
+    "build": "cp -r lib dist && babel --out-dir=dist index.js 'lib/**/*.js'",
```

Comment:
actually, believe it or not, it _doesn't_. when used in an npm script, it will check node_modules/.bin/

```bash
cdieringer@Snapper-osx:~/node/nodeapi$ ls node_modules/.bin/
babel			babel-external-helpers	babel-node		babel-plugin		btoa			knex
```

comes with babel!